config/esp32/components/esp32_mbedtls/empty_file.c

```diff
@@ -0,0 +1 @@
+// Empty file
```

Comment:
Why not remove the entire `esp32_mbedtls` folder?

Reply:
I keep this component to avoid the CI failure on shell example building and QEMU. Add a TODO tag here.

Reply:
Done